tests/integration/mounts_sshfs.bats

```diff
@@ -393,7 +393,8 @@ function fail_sshfs_bind_flags() {
 	pass_sshfs_bind_flags "nodiratime" "bind"
 	run -0 grep -wq nodiratime <<<"$mnt_flags"
 	# MS_DIRATIME implies MS_RELATIME by default.
-	run -0 grep -wq relatime <<<"$mnt_flags"
+	# XXX: rata. For some reason this can't work on my debian system.
+	# run -0 grep -wq relatime <<<"$mnt_flags"
```

Comment:
I'd like people to weight in here, is it safe to remove this grep?

I guess some simple reason might be the cause (like when it is used for the $SOMETHING, then relatime is not shown, or is maybe implied but not shown) but I don't really know.

Reply:
No, this needs to be kept. `diratime` is not part of the strict-rel-no-`atime` enum and it's behaviour is quite specific. This test ensures that a user requesting `nodiratime` doesn't clear `relatime`.

Does this part not work on Debian? `relatime` should be shown in the output of `mount`... An alternative patch would be to verify that everything other than `relatime` is not set.

Reply:
No, it doesn't work here nor with a stable kernel nor testing kernel, using different versions of mount (2.40 is the latest I tried).

I've changed this to check that no other setting that clears relatime is set. Please have a look